### PR TITLE
Project pages

### PR DESCRIPTION
**Pull request by Stereopoly**
* Add individual project pages
* Add main project page that links out
* Update home page

### CODE REVIEW

File: apply.html [hunk 1]
```diff
@@ -71,8 +83,9 @@ <h3>Application Process</h3>
 <footer id="footer" class="footer">
   <div class="container">
     <div class="copyright">
-      &copy; Copyright <strong>ASUC OCTO</strong>. All Rights Reserved. 
-      <a href="mailto:technology@asuc.org"><img src="./assets/img/email.svg"></a>
+      &copy; Copyright <strong>ASUC OCTO</strong>. All Rights Reserved.
```
Comment (g-luo) [inline apply.html:86]: Can you just change it to one line like [Copyright ASUC OCTO. All Rights Reserved. Privacy Policy], and make the privacy policy the same color? I feel like the notice is unnecessary / can feel weird to people

File: projects/apiteam.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  <!-- Google Fonts -->
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i|DM+Sans:300,400,500,600,700" rel="stylesheet">
+
+  <link href="../assets/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
+  <link href="../assets/vendor/font-awesome/css/font-awesome.min.css" rel="stylesheet">
+  <link href="../assets/vendor/ionicons/css/ionicons.min.css" rel="stylesheet">
+  <link href="../assets/vendor/aos/aos.css" rel="stylesheet">
+
+  <link href="../assets/css/style.css" rel="stylesheet">
+  <!-- Global site tag (gtag.js) - Google Analytics -->
```
Comment (g-luo) [inline projects/apiteam.html:19]: maybe save this script in an external js file and import it; it's not a big deal (eg feel free to ignore) but could compress the code a little

**Comment by g-luo** [comment]
> Is the logo square?

does the logo look wonky? I can make it more square

File: apply.html (hunk 1 above)
Comment (Stereopoly) [reply apply.html:86]: I put that statement in cause its technically required to have smt like that but I didn't want to add an entire page

**Comment by Stereopoly** [comment]
image
Doesn't seem quite lined up

File: apply.html (hunk 1 above)
Comment (g-luo) [reply apply.html:86]: we can keep it for now, although imo just putting a link to the privacy policy is sufficient. The language of google analytics seems to say that it just has to have a notice in the privacy policy (https://support.google.com/analytics/answer/2700409?hl=en) and most sites that use it keep the policy fairly discreet / dont have it listed at all.

**Comment by g-luo** [comment]
> image
> Doesn't seem quite lined up

Screen Shot 2020-08-14 at 9 25 56 PM

The sizing looks fine to me; is there a specific width / height the image should be at?

**Comment by Stereopoly** [comment]
* Image issue was an issue with my checked out copy
* Removed detailed footer text leaving just privacy policy link

**Pull request merged**